### DISCUSSION TITLE
线条颜色可以设置为渐变色么

### DISCUSSION THREAD

**Discussion by avrinfly** [hizzgdev/jsmind#570] [General]
需求要求线条颜色不是纯色，是渐变色，请问有字段可以设置吗？

**Comment by hizzgdev** [comment]
目前没有此功能，你可能需要自己修改源码来实现。

**Reply by avrinfly** [reply]
请问如何才能修改源码，并保证后续npm安装的时候，也能生效呢？

**Reply by hizzgdev** [reply]
如果你希望这个功能能够进入 jsmind 的 npm 仓库的话，可以参考[这个文档](https://github.com/hizzgdev/jsmind/blob/master/docs/zh/4.contribution.md)给 jsmind 提交 pull-request

你也可以把代码 clone 到自己的电脑，修改之后放到自己项目里使用，或者发布到自己的 npm package 里，这都是可以的。

**Reply by hizzgdev** [reply]
对于这个功能，如果你有意提交 Pull Request 的话，建议通过扩展 option 里的颜色设置来实现 [view.line_color](https://github.com/hizzgdev/jsmind/blob/master/docs/zh/2.options.md) . 

无论你是否决定提交代码，都非常感谢你的支持！
